### PR TITLE
✨Add google analytics linker config

### PR DESCRIPTION
**Pull request by calebcordry**
Initial config to be added to later.

### CODE REVIEW

**Comment by lannka** [comment]
Adding GA folks to confirm. @zikas @btownsend @avimehta

File: extensions/amp-analytics/0.1/vendors.js [hunk 1]
```diff
@@ -963,6 +963,13 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
       'metric': 'cm',
     },
     'optout': '_gaUserPrefs.ioo',
+    'linkers': {
+      '_gl': {
+        'ids': {
+          '_ga': 'CLIENT_ID(_ga)',
```
Comment (lannka) [inline extensions/amp-analytics/0.1/vendors.js]: oh, this should actually be `CLIENT_ID(AMP_ECID_GOOGLE,,_ga)`

Comment (lannka) [reply extensions/amp-analytics/0.1/vendors.js]: `${clientId}` is even better

Comment (calebcordry) [reply extensions/amp-analytics/0.1/vendors.js]: Done.

File: extensions/amp-analytics/0.1/vendors.js [hunk 2]
```diff
@@ -963,6 +963,13 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
       'metric': 'cm',
     },
     'optout': '_gaUserPrefs.ioo',
+    'linkers': {
+      '_gl': {
+        'ids': {
+          '_ga': '${clientId(AMP_ECID_GOOGLE,,_ga)}',
```
Comment (lannka) [inline extensions/amp-analytics/0.1/vendors.js]: should just be `${clientId}` w/o params.

**Comment by lannka** [comment]
use #17965 instead

**Pull request closed without merging**